### PR TITLE
Initial pass at javabridge/bioformats conda packages

### PR DESCRIPTION
**Pull request by zacsimile**
Works on my mac. Used the install instructions that have historically worked for me on windows. Two more untested things

- Not sure it finds openjdk jre (though it theoretically should) on windows
- If there is another JDK installed, there's no guarantee it installs against openjdk. The user won't notice a difference, but this probably bad form. I think this could easily be solved by setting JAVA_HOME = conda_env_path/bin/java, but I don't know and have not yet found via google what the environment variable for conda_env_path is that I can use in the build.sh/build.bats

### CODE REVIEW

File: python-bioformats/meta.yaml [hunk 1]
```diff
@@ -30,15 +30,21 @@ requirements:
   build:
     - python
     - setuptools
-    - javabridge >=1.0
+    - boto3 >=1.14.23
```
Comment (David-Baddeley) [inline python-bioformats/meta.yaml:33]: surely we need `openjdk` as a build dep too (to ensure that we build and find the right java)?

Comment (David-Baddeley) [reply python-bioformats/meta.yaml:33]: actually, we can probably drop openjdk from both build and run dependencies as it should be pulled in by the python-javabridge package.

File: python-javabridge/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package:
+  name: python-javabridge
+  version: "4.0.0"
+
+source:
+  fn: python-javabridge-4.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/58/5c/bfcc52002c9dfe5fe3bd067783243ab0130f59f757b0517f62f1ba7e88ea/python-javabridge-4.0.0.tar.gz
+  md5: d2cb87302155901c370bc3fa6e58edd2
+
+requirements:  # there are many more requirements (boto3, botocore, future jmespath, numpy, python-dateutil, python-javabridge, s3transfer, six, urllib3) but we bank on python-bioformats pulling these for us
```
Comment (David-Baddeley) [inline python-javabridge/meta.yaml]: requirements should be specified where they are needed so there is a clear dependency chain - this package should stand alone and shouldn't depend on python-bioformats

File: python-bioformats/meta.yaml (hunk 1 above)
Comment (zacsimile) [reply python-bioformats/meta.yaml:33]: It is needed as a build for`python-javabridge`, but there's no java involved in the `python-bioformats` build. It is needed in run, but you're right, `python-javabridge` should take care of it.

File: python-javabridge/meta.yaml (hunk 2 above)
Comment (zacsimile) [reply python-javabridge/meta.yaml]: I thought I'd deleted this comment. It was from an earlier thing I was testing.

**Pull request merged**